.travis.yml

```diff
@@ -1,10 +1,25 @@
+# Copyright 2017 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
 matrix:
   include:
     - language: node_js
       node_js:
         - "node"
 
     - language: python
-      python: 2.7
+      python: 3.4
```

Comment:
Moving to Python 3 is tangentially related to running WPT since I rewrote the new running script in Python 3.